documentation/installing-sos-windows-instructions.md

```diff
@@ -0,0 +1,80 @@
+Installing SOS on Windows
+=========================
+
+SOS will automatically be loaded from the internal Microsoft extension gallery. You need at least version 10.0.18317.1001 or greater of the Windows debugger (windbg or cdb). SOS will load when the "coreclr.dll" module is loaded.
+
+    "C:\Program Files\Debugging Tools for Windows (x64)\cdb.exe" dotnet SymbolTestApp2.dll
+    
+    Microsoft (R) Windows Debugger Version 10.0.18317.1001 AMD64
+    Copyright (c) Microsoft Corporation. All rights reserved.
+
+    0:000> sxe ld coreclr
+    0:000> g
+    ModLoad: 00007ffe`e9100000 00007ffe`e9165000   C:\Program Files\dotnet\host\fxr\2.2.2\hostfxr.dll
+    ModLoad: 00007ffe`e7ba0000 00007ffe`e7c32000   C:\Program Files\dotnet\shared\Microsoft.NETCore.App\2.1.6\hostpolicy.dll
+    ModLoad: 00007ffe`abb60000 00007ffe`ac125000   C:\Program Files\dotnet\shared\Microsoft.NETCore.App\2.1.6\coreclr.dll
+    ntdll!ZwMapViewOfSection+0x14:
+    00007fff`16e2fb74 c3              ret
+    0:000> .chain
+    Extension DLL search Path:
+        C:\Program Files\Debugging Tools for Windows (x64);...
+    Extension DLL chain:
+        sos: image 1.0.1-dev.19106.2+58b97f128be8f866a08aba9fd5c77571ae8e3f6a, API 2.0.0, built Wed Feb  6 13:06:38 2019
+            [path: C:\Users\mikem\AppData\Local\DBG\ExtRepository\EG\cache2\Packages\SOS\1.0.1.0\x64\sos.dll]
+        dbghelp: image 10.0.18317.1001, API 10.0.6,
+            [path: C:\Program Files\Debugging Tools for Windows (x64)\dbghelp.dll]
+        ...
+        ntsdexts: image 10.0.18317.1001, API 1.0.0,
+            [path: C:\Program Files\Debugging Tools for Windows (x64)\WINXP\ntsdexts.dll]
+    0:000> !soshelp
+    -------------------------------------------------------------------------------
+    SOS is a debugger extension DLL designed to aid in the debugging of managed
+    programs. Functions are listed by category, then roughly in order of
+    importance. Shortcut names for popular functions are listed in parenthesis.
+    Type "!help <functionname>" for detailed info on that function.
+
+    Object Inspection                  Examining code and stacks
+    -----------------------------      -----------------------------
+    DumpObj (do)                       Threads (clrthreads)
+    DumpArray (da)                     ThreadState
+    DumpAsync                          IP2MD
+    DumpDelegate                       U
+    DumpStackObjects (dso)             DumpStack
+    DumpHeap                           EEStack
+    ...
+
+## Using the SOS installer global tool
```

Comment:
I suggest naming the sections based on the version of windbg the user wants to use. That is probably the choice that matters to them rather than a decision that they explicitly want to use the global tool. If we get people that want to use the global tool on new versions for some specific reason we can always amend the directions in the future.

Reply:
Btw (entirely fine if you want to fix this later)

Reply:
I ended up changing this section with hopefully better directions for older debuggers.